src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -282,6 +283,8 @@ public void run() {
         } else {
             Logger.v(TAG, "Reuse webview");
         }
+
+        mHelper = new StorageHelper(getApplicationContext());
```

Comment:
AuthenticationActivity could be called in two mode, with broker or in local flow. If it's in local flow, we then initialize the  variable, but no one is using it.